test/index.js

```diff
@@ -34,27 +33,125 @@ tape('init', function (t) {
+
+    var parsed = JSON.parse(res.body.toString())
+    t.deepEqual(parsed, {
+      message: 'jwt must be provided',
+      name: 'JsonWebTokenError',
+      statusCode: 401
+    }, 'should have correct error')
+    t.end()
+  })
+})
+
+tape('should handle \'TokenExpiredError\'', function (t) {
+  var payload = { email: 'chet@scalehaus.io' }
+  var soonToExpireToken = jwt.sign(payload, privateKey, { algorithm: 'RS256', expiresIn: '1' })
+  var opts = {
+    method: 'GET',
+    headers: {
+      Authorization: 'Bearer ' + soonToExpireToken
+    }
+  }
+  var serviceInstance = createService(auth)
+  setTimeout(function test () {
+    servertest(serviceInstance, '/', opts, function (err, res) {
+      t.ifErr(err, 'should not error on expired jwt')
+
+      var parsed = JSON.parse(res.body.toString())
+      t.equal(parsed.statusCode, 401, 'status code matches')
+      t.equal(parsed.message, 'jwt expired', 'should have correct message')
+      t.equal(parsed.name, 'TokenExpiredError', 'should have correct name')
+      t.end()
+    })
+  }, 5) // token expired ~= 4ms ago
+})
+
+tape('should handle \'NotBeforeError\'', function (t) {
+  var nbf = new Date().getTime() + 10000
+  var payload = { email: 'chet@scalehaus.io', nbf }
+  var soonToExpireToken = jwt.sign(payload, privateKey, { algorithm: 'RS256' })
+  var opts = {
+    method: 'GET',
+    headers: {
+      Authorization: 'Bearer ' + soonToExpireToken
+    }
+  }
+  var serviceInstance = createService(auth)
+  setTimeout(function test () {
+    servertest(serviceInstance, '/', opts, function (err, res) {
+      t.ifErr(err, 'should not error on expired jwt')
+
+      var parsed = JSON.parse(res.body.toString())
+      t.equal(parsed.statusCode, 401, 'status code matches')
+      t.equal(parsed.message, 'jwt not active', 'should have correct message')
+      t.equal(parsed.name, 'NotBeforeError', 'should have correct name')
+      t.end()
+    })
+  }, 5)
+})
+
 tape('should handle auth token', function (t) {
-  var opts = {method: 'GET', headers: {
-    Authorization: 'Bearer ' + token
-  }}
+  var now = new Date().getTime()
+  var month = '30 days'
+  var expiresInMs = now + ms(month)
+  var payload = { email: 'chet@scalehaus.io' }
+  // https://github.com/auth0/node-jsonwebtoken#token-expiration-exp-claim
+  var token = jwt.sign(payload, privateKey, { algorithm: 'RS256', expiresIn: month })
+
+  var opts = {
+    method: 'GET',
+    headers: {
+      Authorization: 'Bearer ' + token
+    }
+  }
 
   servertest(createService(auth), '/', opts, function (err, res) {
     t.ifError(err, 'should not error')
     var data = JSON.parse(res.body)
 
     t.equal(data.email, 'chet@scalehaus.io', 'should have correct email')
-    t.equal(data.expiresIn, '30d', 'should have correct expiresIn')
-    t.ok(data.iat, 'chet@scalehaus.io', 'should have iat')
+    t.equal(data.exp, ~~(expiresInMs / 1000), 'should have correct exp claim')
+    t.equal(data.iat, ~~(now / 1000), 'should have correct iat')
```

Comment:
It's been awhile since I've worked with this, so I don't know off-hand about `expiresIn`, but a valid test shouldn't be changed without good reason.

L57 is probably a copy-paste artifact, the second/email argument is a mistake.